rfcs/text/0000-zendesk-triage-sop.md

```diff
@@ -0,0 +1,153 @@
+### DCP PR:
+***Leave this blank until the RFC is approved then the Author(s) must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc\#\](https://github.com/HumanCellAtlas/dcp-community/pull/\<PR\#\>)`
+
+# Technical Support SOP
+
+## Summary
+
+This process RFC describes how the DCP handles technical support requests through the use of the Zendesk application and other processes.
+
+## Author(s)
+
+`[Kevin Osborn](mailto:kosborn2@ucsc.edu)`
+
+`[Jodi Hirschman](mailto:jhirsch@broadinstitute.org)`
+
+`[Matthew Spier](mailto:mspeir@ucsc.edu)`
+
+`[Mallory Freeberg](mailto:mfreeberg@ebi.ac.uk)`
+
+## Shepherd
+
+***Leave this blank. This role is assigned by DCP PM to guide the Author(s) through the RFC process.*
+
+## Motivation
+
+Beta 2 testing will begin soon and we’ve decided to collect feedback
+through our Zendesk ticketing system. We therefore need to quickly
+determine a process for handling larger numbers of Zendesk tickets than
+we have received to date. Several of us have been thinking about this
+for awhile and drew up an initial SOP that we refined at the last DCP
+F2F (Feb. 2019); we present it here as this RFC. 
+
+### **User Stories**
+
+1.  > As a researcher new to the DCP, I would like to get a timely
+    > answer to my question so that I can have a successful experience
+    > submitting, exploring, downloading, or analyzing HCA data.
+
+2.  > As a DCP PM, PO or PL, I would like to be assured that Zendesk
+    > tickets are being responded to in a timely manner so that I can
+    > focus on my own tasks without having to micromanage responses.
+
+3.  > As a DCP team member, I would like to know whether or not I should
+    > be responding to a Zendesk ticket so that I can either respond in
+    > a timely manner or be assured that someone else is responding in a
+    > timely manner.
+
+## Scientific "guardrails" \[optional\]
+
+
+## **Detailed Design**
+
+This section describes the flow of an issue through the support state
+diagram presented below.
+
+![](../images/0000-Zendesk-triage-SOP-diagram.jpeg)
+
+## Issue Creation
+
+Issues in Zendesk originate from users of the DCP. These users could
+include data contributors, researchers trying to use the data in the
+DCP, software developers trying to leverage the DCP API. Any user of the
+DCP can file an issue. Problems identified from the internal DCP staff
+can be filed on Zendesk too, or directly in the applicable development
+repository if the reporter is sure of which one to use. Once an issue is
+created in Zendesk there is a tiered support network that kicks in to
+resolve the question.
+
```

Comment:
It would be good to make a decision between DCP staff members using zendesk or filing directly in an appropriate repo. This sort of if/else policy is likely to create confusion

Reply:
This might be worth some discussion.  If DCP staff members file issues via Zendesk, that helps the triage staff know that an issue is known about and being addressed. (That can be useful if users write about the same issue.) On the other hand, the workload for triage staff increases, and they need to do something with that ticket. Thoughts?

Reply:
> If DCP staff members file issues via Zendesk, that helps the triage staff know that an issue is known about and being addressed.

This is a good point. I think this would fall under the triage staff person "searching the knowledge base" for the solution. One of the places the person should search is in GitHub issues, seeing if a similar issue was already reported and/or solved. 

If we are already understaffed for ZenDesk, then you're right @jodihir we don't want to overburden with tickets from our own DCP staff.

Reply:
Edited